src/ptr/shared.zig

```diff
@@ -232,6 +238,12 @@ pub fn WithOptions(comptime Pointer: type, comptime options: Options) type {
         fn getData(self: Self) if (info.isOptional()) ?*Data else *Data {
             return .fromValuePtr(self.#pointer);
         }
+
+        /// Assumes that the pointer is valid and was created by Shared.new().
+        pub fn cloneFromRawUnsafe(pointer: Pointer) Self {
+            var raw: Self = .{ .#pointer = pointer };
+            return raw.clone();
```

Comment:
```suggestion
            const temp: Self = .{ .#pointer = pointer };
            return temp.clone();
```

`raw` isn't a raw pointer so it should be named something else. Also the `var` is unnecessary